pkgs/top-level/all-packages.nix

```
@@ -1535,6 +1535,8 @@ in
 
   gitless = callPackage ../applications/version-management/gitless { python = python3; };
 
+  gitlint = python3Packages.callPackage ../tools/misc/gitlint { };
```

This should go into `pkgs/top-level/python-packages.nix`

---

That then makes it only available through python3Packages.gitlint. Since its not a python library shouldn't it stay in all-packages? If not, my bad, I'm not very experienced with the python ecosystem.

---

I guess you could optionally add
```suggestion
  gitlint = python3Packages.gitlint;
```
but the primary package definition should be in python-packages.nix.

---

It should be either https://www.github.com/NixOS/nixpkgs/blob/4b4abd7b30c14912486e102a9c2c05d86991e4ee/pkgs/top-level/all-packages.nix#L762 or use buildPythonApplication.